internal/pkg/cli/deploy/workload.go

```diff
@@ -37,13 +39,16 @@ import (
 	"github.com/aws/copilot-cli/internal/pkg/term/log"
 	termprogress "github.com/aws/copilot-cli/internal/pkg/term/progress"
 	"github.com/aws/copilot-cli/internal/pkg/workspace"
+	"github.com/google/uuid"
 	"github.com/spf13/afero"
+	"golang.org/x/sync/errgroup"
 )
 
 const (
 	fmtForceUpdateSvcStart    = "Forcing an update for service %s from environment %s"
 	fmtForceUpdateSvcFailed   = "Failed to force an update for service %s from environment %s: %v.\n"
 	fmtForceUpdateSvcComplete = "Forced an update for service %s from environment %s.\n"
+	waitForImageBuildAndPush  = 100 * time.Second
```

Comment:
I'm just curious how you decided on this amount of time. 🤷🏼

Reply:
I initially thought given an x amount of time for building and pushing images. But the build process the size of dockerfile, so removed the timeout set with `context`.